app/src/main/res/values/strings.xml

```diff
@@ -1327,7 +1327,7 @@ If there are no signs along the whole street which apply for the highlighted sec
 
     <string name="quest_pedestrian_crossing_island">"Does this crossing have an island?"</string>
 
-    <string name="quest_pedestrian_crossing_markings">"Does this crossing have markings on the ground?"</string>
+    <string name="quest_pedestrian_crossing_markings">"Does this crossing have markings that cross the roadway?"</string>
```

Comment:
To my ears, this sounds weird. When I think of that something "crosses the street", I think of people or animals actually going from one side to the other. I don't think it can be used in the context of something that is laid out across (from one end to the other) the street.
Didn't you like my suggestion?

Reply:
E.g. bandits that want to rob oncoming vehicles will put a log **across** the road. But the log doesn't cross the road. It doesn't have legs.

Reply:
Well no, it should have been the same as in "app/src/main/res/values-en/strings.xml".

I think that's just written wrong from memory. My bad.